examples/rvv_memcpy.c

```diff
@@ -27,4 +29,4 @@ int main() {
 
   // compare
   puts(compare_1d(golden, actual, N) ? "pass" : "fail");
-}
```

Comment:
May I ask why is this line updated?

Reply:
In fact, I don't know.I just use vim to edit.

Reply:
It seems that there is a newline character which auto apeend in my file. Although I try to press `delete` at the end.

Reply:
Please remove it as this is not related to the topic of this PR. Coding style issues shall be dealt in a separately.

Reply:
Done.